frontend/src/app/components/ExperimentPage.js

```diff
@@ -95,6 +95,42 @@ export default class ExperimentPage extends React.Component {
     return typeof min === 'undefined' || version >= min;
   }
 
+  getIncompatibleInstalled(incompatible) {
+    if (!incompatible || !this.props.installedAddons.length) {
+      return null;
+    }
+    return Object.keys(incompatible).filter(guid => (
+      this.props.installedAddons.indexOf(guid) !== -1
+    ));
+  }
+
+  renderIncompatibleAddons() {
+    const { incompatible } = this.state.experiment;
+    if (!incompatible) return null;
+    const installed = this.getIncompatibleInstalled(incompatible);
+    if (!installed) return null;
```

Comment:
This needs something like a `|| installed.length === 0` otherwise I see this:

<img width="675" alt="screen shot 2016-09-22 at 11 32 37 pm" src="https://cloud.githubusercontent.com/assets/87619/18776337/66bb8ef0-811d-11e6-878f-91568a18e690.png">